### ISSUE TITLE
verification plan-beam technique error

### ISSUE THREAD

**Issue by patm95** [VarianAPIs/Varian-Code-Samples#57]
Hello all.  Our verification plan script has been running pretty well.  However, we periodically get an error:  "The Beam technique does not match the technique in BeamParameters".  Has anyone ran into this?  We only use 6X flattened beam for IMRT here and only have one machine.  For one particular patient, we got this error on the initial plan, but not the boost.  In every case, we can create a verification plan manually and it QAs fine.

**Comment by avalgoma** [comment]
Hi.
I also get the same error in the verification plans using the script. Truebeam 6x, 6fff, 10fff, 15x is our configuration, but not sure if the error occurs for all the energies. I have to check that point.
Regards.

**Comment by patm95** [comment]
Interesting.  For what it's worth our machine is a clinac iX.  Glad to know
that I am not the only one with this problem.  If no one has found a
solution to this, I may start poking around on this in my spare time.

**Comment by joecastelo** [comment]
Guys, maybe the code being so simple is leading to some problems.
I did some separation of concerns, and it is working fine.
I step into this technique error in #55 , maybe it has to do with the last tag (fluencemodeid)

**Comment by joecastelo** [comment]
But I only tried with NovalisTX 6X 6X-SRS and TrueBeam 6X-FFF 6X

**Comment by patm95** [comment]
Thank you Joao.  I will try this and report back.

**Comment by joecastelo** [comment]
@patm95 and @avalgoma 

Do you guys tried to add beams with dose rate different than 600 ? 
When I add any beam they always come with 600 MU/min independent wheter I choose the right externalmachineparameters

**Comment by joecastelo** [comment]
Our FFF has only 1400 DRate, and it somehow changes to 600. and this dose rate does not even exists....

**Comment by joecastelo** [comment]
The same happens for our 6X SRS with dose rate 1000, maybe your errors are being caused becaused of the different dose rate? Does halcyon has a 600 MU/min Dose Rate.

If I try to add the beam for 6X-FFF or SRS with dose rate 600 it wont let me, but i put 1400 or 1000 and suddenly after creation it turns into 600 MU/min. Man I just can't believe. Tell me I'm crazy hahaha

**Comment by avalgoma** [comment]
Yes, Halcyon has a 600 MU/min Dose Rate. 

However, imagine a plan in which control points indicate 600 UM/min. Well, the treated plan has nothing to do with those 600 UM/min.

It appeas that the important point from the Halcyon point of view is MU/degree, not UM/min nor gantry speed.

-- Agus

**Comment by joecastelo** [comment]
Yeah it does make sense, the MU/degree are always right, but the dose rate problem is tricky. The beam has its dose rate, defining the max dose rate. Regarding VMAT is OK but CRT makes no sense that dose rate is not right.

**Comment by patm95** [comment]
We always use the 600 rep rate.  And always use 6x flattened beam. I still
have not identified what exactly was causing our error.

**Comment by joecastelo** [comment]
Did you try to create beams with other dose rates? Just to check if is only my problem

**Comment by patm95** [comment]
I just tested with other dose rates on my problem plan.  It would create
the verification plan on other dose rates, but not with a dose rate of
600.  I basically just copied the plan and modified the dose rate and reran
the script.  Very odd.

**Comment by joecastelo** [comment]
But are you managing to change the beam dose rate??

**Comment by patm95** [comment]
Oh no I am not trying to change the dose rate.  Just basically copying the
beam parameters from the clinical beam to the QA beam.

**Comment by leylaMoghaddasi** [comment]
Has anyone found a solution for DoseRate being changed to 600MU/min when copied across to verification plan?
